fastlane/lib/fastlane/fast_file.rb

```diff
@@ -1,3 +1,26 @@
+require "rubygems"
+
+class String
```

Comment:
Those extensions should probably be moved somewhere else (not `fast_file.rb`), we have a `core_ext` directory where we can put it. One file per class 👍